src/StemEcosystem.Web.pyproj

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How long does it take to build these Visual Studio files? I hesitate to include local config files in the project for fear of muddying things up.

Reply:
The .sln and .pyproj files shouldn't affect any build.. hopefully. If someone comes along and wants to use Visual Studio (like me), all they have to do is click the .sln file to open and run the project, otherwise I can fork and use that fork for personal settings if we want to keep things kosher.

Reply:
Sure, that makes sense. That's why we use `.gitignore` to remove `.vscode` JSON config files. It certainly doesn't hurt anything but it keeps out config conflicts between devs and keeps the repo clean. I don't want to be the only one making that decision so whatever the group thinks is cool with me!

Reply:
I think I would prefer to keep those files out of the repo as well.